src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -267,11 +266,19 @@ public void RunTests(TestRunRequestPayload testRunRequestPayload, ITestHostLaunc
                 this.LogTelemetryForLegacySettings(requestData, runsettings);
             }
 
-            if (!commandLineOptions.IsDesignMode)
+            // get Fakes data collector settings
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("_Disable_Fakes_DataCollector_Configuration")))
```

Comment:
Who would use this? We usually name env variables like this VSTEST_<component>_<action> (e.g. VSTEST_RUNNER_DEBUG`,  and spell them in uppercase, in case some OSes would consider spelling when grabbing the variable. We also consider 0 to be off, not just null.

Reply:
This is for the users in the vstest task. By Shreyas' suggestion, if something is being enabled that wasn't before, it's best to have a fallback to be able to replicate previous behavior. I can change the var name and behavior to meet the standard in the vstest repo